### ISSUE TITLE
Incomplete backport of https://github.com/oracle/graal/pull/3329/ to release/graal-vm/20.3 

### ISSUE THREAD

**Issue by zakkak** [oracle/graal#3591]
**Describe the issue**
It looks like 46086187bbda793bb92fae2d5f34f12075ff2914 was left out when backporting https://github.com/oracle/graal/pull/3329/ to branch `release/graal-vm/20.3 `

This results in the Native image bash launcher printing:
```
dev (Java Version 1.0.12+5-jvmci-20.3-b17)
```
instead of
```
native-image dev Java 11 CE (Java Version 1.0.12+5-jvmci-20.3-b17)
```

**Steps to reproduce the issue**
```
git clone --depth 1 --branch release/graal-vm/20.3  git@github.com:oracle/graal.git
cd graal
mx --primary-suite-path substratevm --components='Native Image' build 
./sdk/latest_graalvm_home/bin/native-image --version
```

**Describe GraalVM and your environment:**
 - GraalVM version: **dbf1542c496eaffffe82b2edf4881214738b885e**
 - JDK major version: **11**
 - OS: **Fedora 34**
 - Architecture: **AMD64**

**Comment by zakkak** [comment]
@marwan-hallaoui can you please have a look? (For some reason I can't assign this to you)

**Comment by marwan-hallaoui** [comment]
@zakkak in fact the requested changes are not yet backported to branch release/graal-vm/20.3, I created a task to do so the fix will be available for 20.3.4. 
Thanks

**Comment by zakkak** [comment]
This has been merge in 20.3.4 (see https://github.com/oracle/graal/commit/dd88f0579d76f7bb87d38416da9b40d4acb8d56b)
